workflows/VGP-assembly-v2/Scaffolding-HiC-VGP8/Scaffolding-HiC-VGP8-tests.yml

```diff
@@ -16,12 +16,27 @@
       class: File
       location: https://zenodo.org/records/10037496/files/Estimated%20genome%20size%20-%20Parameter%20File.txt?download=1
       filetype: txt
+    Sequence graph (optional): null
```

Comment:
Please stop adding those optional labels, I keep removing them and you keep adding them. They're just redundant.